server/server.go

```diff
@@ -4223,3 +4223,29 @@ func (s *Server) changeRateLimitLogInterval(d time.Duration) {
 	default:
 	}
 }
+
+// Disconnects a client by cid
+func (s *Server) DisconnectClientByID(id uint64) error {
```

Comment:
Should it be CID to be consistent with that meaning connection ID?

```suggestion
func (s *Server) DisconnectClientByCID(id uint64) error {
```